lib/yubihsm.h

```diff
@@ -554,7 +554,8 @@ typedef struct {
   uint8_t sequence;
   /// Object origin
   uint8_t origin;
-  /// Object label. The label consists of raw bytes and is not restricted to printable characters or valid UTF-8 glyphs
+  /// Object label. The label consists of raw bytes and is not restricted to
```

Comment:
While it's potentially good to let clang-format fix this it doesn't really belong with the rest of this, I wouldn't add this but push it to the future.

Reply:
Not sure I can do much about it without changing pre-commit setup. Commit fails without it